packages/medusa/src/utils/get-query-config.ts

```diff
@@ -60,15 +60,17 @@ export function prepareListQuery<T extends RequestQueryFields, TEntity>(
         return !(
           field.startsWith("-") ||
           field.startsWith("+") ||
+          field.startsWith(" ") ||
           field.startsWith("*")
         )
       })
+
     if (shouldReplaceDefaultFields) {
-      allFields = new Set(customFields.map((f) => f.replace(/^[+-]/, "")))
+      allFields = new Set(customFields.map((f) => f.replace(/^[+ -]/, "")))
     } else {
       customFields.forEach((field) => {
-        if (field.startsWith("+")) {
-          allFields.add(field.replace(/^\+/, ""))
+        if (field.startsWith("+") || field.startsWith(" ")) {
```

Comment:
Q: why are we checking for empty space here?

Reply:
because most clients will transform `+` into and empty space, and that is what Medusa app will receive.